### ISSUE TITLE
Create a documentation for the API

### ISSUE THREAD

**Issue by heitorsilva** [gridsound/api.gridsound.com#5]
It would be nice to have a documentation, accessible via a webpage, to understand how the API works, and to make it easier for other developers to contribute.

**Comment by mr21** [comment]
Indeed, after our discussion on discord, i think the solution should be storable directly on this repo into the `doc/` file to be accessible at http://gridsound.github.io/api.gridsound.com.

Or just some markdown on the wiki of this repo, idk yet
